runtimes/peregrine/src/lib.rs

```diff
@@ -936,29 +936,27 @@ construct_runtime! {
 		Authorship: pallet_authorship::{Pallet, Storage} = 20,
 		AuraExt: cumulus_pallet_aura_ext = 24,
 
-		// Governance stuff
 		Democracy: pallet_democracy = 30,
 		Council: pallet_collective::<Instance1> = 31,
 		TechnicalCommittee: pallet_collective::<Instance2> = 32,
-		// placeholder: parachain council election = 33,
+		// Placeholder: parachain council election = 33,
```

Comment:
Do we need this here? Cc @weichweich

Reply:
yeah at some point we want to have another election strategy. index 33 is reserved for that. So maybe replace `placeholder` with `reserved`.